src/hooks/useAuth.ts

```diff
@@ -19,7 +28,21 @@ const useAuth = () => {
             activate(connector)
           }
         } else {
-          toastError(error.name, error.message)
+          window.localStorage.removeItem(connectorLocalStorageKey)
+          if (error instanceof NoEthereumProviderError || error instanceof NoBscProviderError) {
+            toastError('Provider Error', 'No provider was found')
+          } else if (
+            error instanceof UserRejectedRequestErrorInjected ||
+            error instanceof UserRejectedRequestErrorWalletConnect
+          ) {
+            if (connector instanceof WalletConnectConnector) {
+              const walletConnector = connector as WalletConnectConnector
+              walletConnector.walletConnectProvider = null
```

Comment:
what this line is for ?

Reply:
This fixes the issue of qr code not shown again after rejecting it. We have to clean the state in WalletConnectConnector otherwise it will not show the qr code again.